fiftyone/server/view.py

```
@@ -791,30 +791,38 @@ def _add_frame_labels_tags(path, field, view):
-        F(_LABEL_TAGS).extend(
-            F("frames").reduce(
-                VALUE.extend((F(tags) != None).if_else(F(tags), [])), []
-            )
-        ),
-        _allow_missing=True,
+    view = view.add_stage(
+        fosg.SetField(
+            _LABEL_TAGS,
+            F(_LABEL_TAGS).extend(
+                F("frames").reduce(
+                    VALUE.extend((F(tags) != None).if_else(F(tags), [])), []
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)